driver/bpf/quirks.h

```diff
@@ -28,8 +28,8 @@ or GPL2.txt for full copies of the license.
 #define BPF_SUPPORTS_RAW_TRACEPOINTS
 #endif
 
-#if defined(__aarch64__) && !defined(BPF_SUPPORTS_RAW_TRACEPOINTS)
-    #error The BPF ARM64 support requires 'raw_tracepoints' so kernel versions greater or equal than '4.17'.
+#if CAPTURE_SCHED_PROC_FORK && !defined(BPF_SUPPORTS_RAW_TRACEPOINTS)
```

Comment:
Hmm, `#if defined(CAPTURE_SCHED_PROC_FORK && !defined(BPF_SUPPORTS_RAW_TRACEPOINTS)`?

Reply:
I think `#ifdef CAPTURE_SCHED_PROC_EXEC && !defined(BPF_SUPPORTS_RAW_TRACEPOINTS)` is not necessary since the `CAPTURE_SCHED_PROC_EXEC` logic could work without raw_tracepoint, only `CAPTURE_SCHED_PROC_FORK` needs raw_tracepoint... i have explained it a little bit here https://github.com/falcosecurity/libs/pull/452/files#diff-e50a25dfdff0bacf40a38fb8cbce7af10e7aeb6222fb55e84ba227ef6829692bR94-R96